utils/autogen/Specific/subcrystal_taub.py

```diff
@@ -1,20 +1,21 @@
 # RunCrystal for Taub
 
-def execute(px_ssh, infile, outfile):
+def execute(px_ssh, infile, outfile, job_name):
   px_ssh.prompt()
 
-  qscript = """#PBS -l nodes=1:ppn=16
+  qscript = """#PBS -l nodes=1:ppn=12
```

Comment:
I'd rather not commit back and forth our different installation stuff. Either we find something that works for both or not.